x-pack/legacy/plugins/siem/public/components/query_bar/index.test.tsx

```diff
@@ -196,9 +189,13 @@ describe('QueryBar ', () => {
 
   describe('#onQueryChange', () => {
     test(' is the only reference that changed when filterQueryDraft props get updated', () => {
+      const KibanaWithStorageProvider = createKibanaContextProviderMock();
```

Comment:
@XavierM this was a fun one! It ultimately ended up being an upstream issue with `SearchBar`: it relies on a 'storage' service which does not exist in new platform nor its mocks, which ultimately meant the query bar wasn't being rendered.

We were previously [mocking this service](https://github.com/elastic/kibana/pull/52783/files#diff-406c4710f953ab79d87b6284213b9877L57-L59), and 9f4f0dda14 continues to do that for this one test that needs it, but I wanted to get your opinion on how to proceed, here. This is related to #53029 in that the test failure was caused by a dependency pretty far removed.

Reply:
/cc @lizozom in case you have thoughts on this one!